omnisend/includes/Internal/class-connection.php

```diff
@@ -45,16 +45,26 @@ private static function get_account_data( $api_key ): array {
 			)
 		);
 
+		if ( is_wp_error( $response ) ) {
+			return array();
+		}
+
 		$body = wp_remote_retrieve_body( $response );
-		if ( ! $body ) {
-			return '';
+
+		if ( empty( $body ) ) {
+			return array();
 		}
 
 		$arr = json_decode( $body, true );
 
-		return is_array( $arr ) ? $arr : array();
+		if ( $arr === null ) {
```

Comment:
I think previous check was better. It was actually checking if $arr is array, if not just returning empty array. With your change you are checking not array, but null. Not sure why. Suggest to return previous check or explain your change.